corrector.rb

```diff
@@ -0,0 +1,10 @@
+class Corrector
+  def correct_name(name)
+    given_name = name.strip.capitalize
+    if given_name.length > 10
+      given_name[0, 10]
+    else
+      given_name
+    end
+  end
+end
```

Comment:
- [ ] good job setting up the `corrector` class 👍 however in the `Person` there should be an instance of Corrector in Person on initialization set.
- [ ] In the `Person` class there should be a method validate_name that uses the Corrector instance to validate and save the corrected name.

Reply:
- [ ] Very nice work addressing the issue raised here. Unfortunately, you forgot to import the `Corrector` class in the `Person` class. That basically means the `Person` class cannot access the `Corrector` class and trying to initialize an object instance of a `Corrector` class will raise an error. Kindly fix this issue. :pray: 

![Screenshot from 2021-11-16 12-41-39](https://user-images.githubusercontent.com/64894927/141961351-879c2d49-7b98-44b5-ba34-2f6130a78cc0.png)